test/test_onnx.py

```diff
@@ -121,6 +121,11 @@ def test_roi_align(self):
         model = ops.RoIAlign((5, 5), 1, 2)
         self.run_model(model, [(x, single_roi)])
 
+        x = torch.rand(1, 1, 10, 10, dtype=torch.float32)
+        single_roi = torch.tensor([[0, 0, 0, 4, 4]], dtype=torch.float32)
+        model = ops.RoIAlign((5, 5), 1, 2, aligned=True)
```

Comment:
Can you test for more configurations of the `roi` (including malformed boxes), and different `scale_factor`?